numpy/_core/numeric.pyi

```diff
@@ -497,39 +497,6 @@ def tensordot(
     axes: int | tuple[_ShapeLike, _ShapeLike] = ...,
 ) -> NDArray[object_]: ...
 
-@overload
-def vecdot(
-    x1: _ArrayLikeUnknown, x2: _ArrayLikeUnknown, axis: int = ...
-) -> NDArray[Any]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeBool_co, x2: _ArrayLikeBool_co, axis: int = ...
-) -> NDArray[np.bool]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeUInt_co, x2: _ArrayLikeUInt_co, axis: int = ...
-) -> NDArray[unsignedinteger[Any]]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeInt_co, x2: _ArrayLikeInt_co, axis: int = ...
-) -> NDArray[signedinteger[Any]]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeFloat_co, x2: _ArrayLikeFloat_co, axis: int = ...
-) -> NDArray[floating[Any]]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeComplex_co, x2: _ArrayLikeComplex_co, axis: int = ...
-) -> NDArray[complexfloating[Any, Any]]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeTD64_co, x2: _ArrayLikeTD64_co, axis: int = ...
-) -> NDArray[timedelta64]: ...
-@overload
-def vecdot(
-    x1: _ArrayLikeObject_co, x2: _ArrayLikeObject_co, axis: int = ...
-) -> NDArray[object_]: ...
```

Comment:
Why can these type annotations be deleted?

Reply:
Ah because vecdot is a ufunc now and ufunc.pyi handles it for us, right?

Reply:
Yes! For example there is no `def matmul(...)` in `pyi` files.